qt-models/diveplannermodel.cpp

```diff
@@ -258,7 +265,15 @@ QVariant DivePlannerPointsModel::data(const QModelIndex &index, int role) const
 			else
 				return p.time / 60;
 		case GAS:
-			return get_gas_string(displayed_dive.cylinder[p.cylinderid].gasmix);
+			/* Check if we have the same gasmix two or more times
+			 * If yes return more verbose string */
+			int same_gas = same_gasmix_cylinder(&displayed_dive.cylinder[p.cylinderid], p.cylinderid, &displayed_dive, true);
+			if (same_gas == -1)
+				return get_gas_string(displayed_dive.cylinder[p.cylinderid].gasmix);
+			else
+				return get_gas_string(displayed_dive.cylinder[p.cylinderid].gasmix) +\
+					QString(" (%1 %2 ").arg(tr("cyl.")).arg(p.cylinderid + 1)\
```

Comment:
no need for the `\` at the end of the lines here. you can have such lines ending with ` +`, so that the next line doesn't start with the `+ `.